src/System Application/App/Email/src/EmailUpgrade.Codeunit.al

```diff
@@ -35,17 +36,44 @@ codeunit 1597 "Email Upgrade"
         UpgradeTag.SetUpgradeTag(GetDefaultEmailViewPolicyUpgradeTag());
     end;
 
+    local procedure SetDefaultEmailMaxConcurrencyLimit()
+    var
+        EmailRateLimit: Record "Email Rate Limit";
+        UpgradeTag: Codeunit "Upgrade Tag";
+        EmailRateLimitImpl: Codeunit "Email Rate Limit Impl.";
+    begin
+        if UpgradeTag.HasUpgradeTag(GetDefaultEmailMaxConcurrencyLimitUpgradeTag()) then
+            exit;
+
+        if EmailRateLimit.FindSet() then
+            repeat
```

Comment:
SetRange, then modify all

Reply:
you could also use the data transfer, however if there are few records in the table, feel free to won't fix this comment.